pkg/dd/monitor.go

```diff
@@ -0,0 +1,54 @@
+package dd
+
+import (
+	"fmt"
+	"strconv"
+
+	aw "github.com/deanishe/awgo"
+	"gopkg.in/zorkian/go-datadog-api.v2"
+)
+
+// Monitor is workflow structure of the Monitor.
+type Monitor struct {
+	client    *datadog.Client
+	cacheName string
+	wf        *aw.Workflow
+}
+
+// NewMonitor returns a new dd.Monitor.
+func NewMonitor(client *datadog.Client, wf *aw.Workflow) Monitor {
+	return Monitor{
+		cacheName: monitorCacheName(),
+		client:    client,
+		wf:        wf,
+	}
+}
+
+// ListDashboards is fetch monitor and appends to workflow items.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [comment on exported method Monitor.ListMonitors should be of the form "ListMonitors ..."](https://app.codacy.com/manual/nekottyo/alfred-datadog-workflow/pullRequest?prid=5421329)